src/test/components/FilterNavigatorBar.test.js

```diff
@@ -0,0 +1,93 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import * as React from 'react';
+import { render } from 'react-testing-library';
+import { Provider } from 'react-redux';
+
+import ProfileFilterNavigator from '../../components/app/ProfileFilterNavigator';
+import * as ProfileView from '../../actions/profile-view';
+import { storeWithProfile } from '../fixtures/stores';
+import { getProfileFromTextSamples } from '../fixtures/profiles/processed-profile';
+
+describe('app/ProfileFilterNavigator', () => {
+  const browsingContextID = 123123;
+  function setup() {
+    const { profile } = getProfileFromTextSamples(`
+      A  A  A
+      B  B  B
+      C  C  H
+      D  F  I
+      E  G
+      `);
+    // Add page for active tab.
+    profile.pages = [
+      {
+        browsingContextID: browsingContextID,
+        innerWindowID: 1,
+        url: 'https://developer.mozilla.org/en-US/',
+        embedderInnerWindowID: 0,
+      },
+    ];
+    profile.meta.configuration = {
+      threads: [],
+      features: [],
+      capacity: 1000000,
+      activeBrowsingContextID: browsingContextID,
+    };
+
+    // Change the root range for testing.
+    const samples = profile.threads[0].samples;
+    samples.time[samples.length - 1] = 50;
+
+    const store = storeWithProfile(profile);
+    const renderResult = render(
+      <Provider store={store}>
+        <ProfileFilterNavigator />
+      </Provider>
+    );
+
+    return {
+      ...store,
+      ...renderResult,
+    };
+  }
+
+  it('renders ProfileFilterNavigator properly', () => {
```

Comment:
Added some basic rendering tests as well for this component. It wasn't tested before. I know we don't like the snapshots too much anymore, but couldn't stop myself :) at least we can see the basic rendering here.

Reply:
Snapshots are fine, as long as they are actually covering things that are useful.